logs_ingest/main.py

```diff
@@ -127,6 +127,11 @@ def parse_record(record: Dict, self_monitoring: SelfMonitoring):
     category = record.get("category", "").lower()
     infer_monitored_entity_id(category, parsed_record)
 
+    attribute_value_length_limit = get_int_environment_value("DYNATRACE_LOG_INGEST_ATTRIBUTE_VALUE_MAX_LENGTH", 250)
+    for attribute_key in parsed_record:
+        if attribute_key not in ["content", "severity", "timestamp"] and parsed_record[attribute_key]:
+            parsed_record[attribute_key] = parsed_record[attribute_key][0: attribute_value_length_limit]
```

Comment:
I think we could make a couple improvements here:
- iterate over item tupples to avoid multiple map accesses
- convert attribute value to `str` in case if any metadata rule attribute returns other type of value
- (minor) 0 is not required in slice 
```suggestion
    for attribute_key, attribute_value in parsed_record.items():
        if attribute_key not in ["content", "severity", "timestamp"] and attribute_value:
            string_attribute_value = attribute_value
            if not isinstance(attribute_value, str):
                string_attribute_value = str(attribute_value)
            parsed_record[attribute_key] = string_attribute_value[: attribute_value_length_limit]
```